src/main/java/com/bravo/payment/model/dto/PaymentReadDto.java

```diff
@@ -1,10 +1,10 @@
-package com.bravo.user.model.dto;
+package com.bravo.payment.model.dto;
 
 import java.time.LocalDateTime;
 import lombok.Data;
 
 @Data
-public class PaymentDto {
+public class PaymentReadDto {
```

Comment:
What makes a Dto a ReadDto?  This naming pattern is new one to me.

Reply:
The linked story/requriements (0014) for this story seemed a bit inconsistent in its ask, but the intention very much seemed to be to store the encrypted full CC number (SaveDto), but read back only the last 4 (ReadDto).  This merits discussion with POs.